components/director/run.sh

```diff
@@ -72,10 +75,10 @@ set -e
 
 echo -e "${GREEN}Populate DB${NC}"
 
-PGPASSWORD=pgsql@12345 psql -U ${DB_USER} -h 127.0.0.1 -f <(cat ${ROOT_PATH}/../schema-migrator/migrations/director/*.up.sql) ${DB_NAME}
-
-PGPASSWORD=pgsql@12345 psql -U ${DB_USER} -h 127.0.0.1 -f <(cat ${ROOT_PATH}/../schema-migrator/seeds/director/*.sql) ${DB_NAME}
-
+cat ${ROOT_PATH}/../schema-migrator/migrations/director/*.up.sql | \
+    docker exec -i ${POSTGRES_CONTAINER} psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}"
+cat ${ROOT_PATH}/../schema-migrator/seeds/director/*.sql | \
+    docker exec -i ${POSTGRES_CONTAINER} psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}"
```

Comment:
I would remove `interactive` mode

Reply:
I've considered that, but I've left it there because of the pipe before the `docker exec`.

https://stackoverflow.com/a/43099210 
>Or you can change it to -t if you want TTY support but don't have it available on the input device. Do this for color formatting of the output in your logs, or for when you later attach to the container with a proper terminal.

Reply:
Yes you're right :)